app/src/main/java/com/github/se/polyfit/data/api/SpoonacularAPI.kt

```diff
@@ -168,4 +165,78 @@ class SpoonacularApiCaller {
       RecipeFromIngredientsResponseAPI.faillure()
     }
   }
+
```

Comment:
Yeah, I 100% agree with you. I should have used `moshi`, but I hadn't discovered it yet, so I implemented the JSON parsing myself. Since I did not modify the previous API call functions, I do not think it is within the scope of this PR to change this.